tbx/project_styleguide/templates/patterns/molecules/listing/listing--avatar.html

```diff
@@ -12,7 +12,7 @@ <h2 class="listing-avatar__title listing__title">{{ title }}</h2>
                     <span class="listing__name mr-1.5">{{ name }}</span>
                 </p>
                 <p class="listing__meta">
-                    <span class="mr-1.5">{{ page_type }},</span> {{ reading_time }}
+                    <span class="mr-1.5">{{ page_type }},</span> {{ reading_time }} min read
```

Comment:
It looks like the `reading_time` and `read_time` vars (latter used on `blog_detail.html`) only output the number, so I needed to add the 'min read' text here.

Reply:
You might also want to use the `pluralize` filter as I did on the blog listing.

Reply:
actually I've just realised I changed this same file - and also added the date, so you might just be able to remove these changes.

Reply:
I'll remove my changes and pick yours up when you merge your branch in that case

Reply:
Reset the changes in `listing--avatar.html` and `listing--image.html`

Reply:
https://github.com/torchbox/torchbox.com/pull/41/commits/6b50c4b6f607c5446e86144dd761b653bc824e75

Reply:
I'll update listing--image.html in my branch as well - had previously only changed listing--avater